### PR TITLE
added more footer stuff

### PR DESCRIPTION
**Pull request by WilsontheWolf**
specifically, dark theme text can be clicked, will be doing something with this soon.
probs something else I forgot about.

### CODE REVIEW

**Comment by WilsontheWolf** [comment]
added better theme button looky

File: www/html/partials/footer.ejs [hunk 1]
```diff
@@ -1,15 +1,19 @@
...
   let check = document.getElementById('darkCheckbox');
+  let label = document.getElementById('darkLabel');
+  let css = document.getElementById('darkCSS');
   check.checked = getCookie('darkTheme') == 'true';
   changeTheme();
   
   function changeTheme() {
-    document.getElementById('darkCSS').disabled = !check.checked;
+    css.disabled = !check.checked;
+    if (check.checked) label.innerText = "🌙 Dark Theme"
```
Comment (WilsontheWolf) [inline www/html/partials/footer.ejs:15]: change to `if (!check.checked)` for inverted behaviour: looky

**Comment by Alexejhero** [comment]
I like how this looks, but I don't think it fits in with the other stuff. Let me do something and i'll show it to you

**Comment by Alexejhero** [comment]
I have something better in mind for this button.

**Pull request closed without merging**